src/RestaurantApi/Users/RemoveUser.js

```diff
@@ -0,0 +1,14 @@
+import { db } from "../../firebase"
+import { doc, deleteDoc } from 'firebase/firestore'
+
+const removeUser = async (id) => {
+  const userDoc = doc(db, "users", id);
+
+  try {
+    await deleteDoc(userDoc);
+  } catch (e) {
+    console.error(e.message, e.name);
+  } 
```

Comment:
throw an error instead of just console it, when tou just console.error it will only appear in console. We want to show the error to the user. It means that when you throw an error here, component which use this function can use try ... catch to catch this error and show it. It is better to throw error here and higher component can just use it instead of creating message